doc/BUILD

```diff
@@ -326,6 +326,37 @@ doctest(
 )
 
 
+doctest(
+    name="doctest[train]",
+    files = glob(
+        include=[
+            "source/train/**/*.rst",
+            "source/train/**/*.md"
+        ],
+        exclude=[
+            # GPU
+            "source/train/user-guides/data-loading-preprocessing.rst",
+            "source/train/user-guides/using-gpus.rst"
+        ]
+    ),
+    tags = ["team:ml"]
+)
+
+doctest(
+    name="data-loading-preprocessing.rst",
+    files = ["source/train/user-guides/data-loading-preprocessing.rst"],
+    tags = ["team:ml"],
+    gpu = True,
+)
+
+doctest(
+    name="using-gpus.rst",
+    files = ["source/train/user-guides/using-gpus.rst"],
+    tags = ["team:ml"],
+    gpu = True,
+)
```

Comment:
Is it possible to merge these two test units into one "doctest[train,gpu]"?

Reply:
We could. GPU tests are usually longer running, and if you're running many of them in a single Bazel test we might get timeouts. For regular tests, we also run one Bazel test per file.

That said, since there's only two GPU files for now, I've merged them into a single Bazel test.